pymeasure/instruments/agilent/agilentB1500.py

```diff
@@ -175,6 +208,30 @@ def force_gnd(self):
         """
         self.write("DZ")
 
+    def restore_settings(self):
+        """Restores the settings of all channels to the state before
```

Comment:
Docstrings should be (according to PEP) in imperative mode:

```suggestion
        """Restore the settings of all channels to the state before
```